### PR TITLE
Remove third parameter from createFormBuilder call

### PR DESCRIPTION
**Pull request by Hocdoc**

### CODE REVIEW

File: components/form/introduction.rst [hunk 1]
```diff
@@ -526,7 +526,7 @@ by ``handleRequest()`` to determine whether a form has been submitted):
 
         public function searchAction()
         {
-            $formBuilder = $this->createFormBuilder('form', null, array(
```
Comment (HeahDude) [inline components/form/introduction.rst]: Hi @Hocdoc, actually you should remove `'form'` instead of `null`

**Comment by Hocdoc** [comment]
Oops, you are right. I've adapted the PR.

**Comment by HeahDude** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
thanks @Hocdoc! I've merged your changes into the 2.3 branch (that's why this PR seems like its closed instead of merged). I'll take care of merging it into the newer versions from there.

**Pull request closed without merging**